mgl32/codegen.go

```diff
@@ -138,10 +142,16 @@ func genMgl64(destPath string) {
 			return err
 		}
 
-		if _, err = io.Copy(out, in); err != nil {
+		src, err := ioutil.ReadAll(in)
+		m64 := strings.Replace(string(src), "github.com/luxengine/math", "math", -1)
+		b := bytes.NewBuffer(([]byte)(m64))
+
+		if _, err = io.Copy(out, b); err != nil {
 			return err
 		}
 
+		out.Close()
```

Comment:
I'm not really proud of those lines to rewrite all `github.com/luxengine/math` to `math` but gofmt cannot rewrite imports